runtime/resolvers/sql.go

```diff
@@ -17,6 +17,8 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
+const sqlResolverInteractiveRowLimit = 10000
```

Comment:
Nit: 
1. Can this be made configurable via some project specific config ? Either via undocumented rill.yaml or admin command. 
2. 10K seems a bit more restrictive, how about increasing the limit to 100K ?

Reply:
This is not for exports, only interactive API requests. For something like 100k rows, for interactive queries, shouldn’t we push people to use pagination? (I.e. make multiple requests and have args like `{{ .args.page_offset }}`.)

The network limit for responses is currently 4MB, so 100k seems quite likely to fail at network level anyway.

If you want, I can make this configurable. I'll do it in a follow-up PR because I'll need to add some new abstractions for it :)

Reply:
sounds good.